### ISSUE TITLE
Document incremental builds

### ISSUE THREAD

**Issue by contra** [gulpjs/gulp#841]
vinyl-fs has support for a new since parameter to only spit out modified files. we should document how to use this (possibly with a registry?) with the new task system

**Comment by boogerlad** [comment]
This sounds like a great idea. If I'm understanding this correctly, the common "clean" task will no longer have to be ran to clear out the destination folder?

**Comment by dinoboff** [comment]
#937 document the `gulp.src` `since` option.

I can document the new api to get the last run once (phated/undertaker#9) once it lands in Gulp.

**Comment by phated** [comment]
Thanks to the efforts by @dinoboff this is implemented and documented in the 4.0 branch.  Closing.
